sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.4.0 (TBD)
+
+- Added inline image support
```

Comment:
The README check in the pipeline might fuss at you if you don't use `(Unreleased)` for the date and preserve the subsections (e.g. bring back `### Features Added`